### ISSUE TITLE
Dummy apps and their Gemfiles

### ISSUE THREAD

**Issue by paneq** [RailsEventStore/rails_event_store#193]
In `bounded_context` dummy apps have Gemfile such as:

```
source 'https://rubygems.org'

gem 'rails', '~> 5.1.4'
gem 'bounded_context', path: '../../'
```

I was wondering why not:

```
source 'https://rubygems.org'

gem 'rails', '5.1.4'
gem 'bounded_context', path: '../../'
```

Why are we using `~` ?

**Comment by paneq** [comment]
Related #192

**Comment by paneq** [comment]
What do you think @pawelpacana ?

**Comment by pawelpacana** [comment]
Makes sense!

**Comment by mpraglowski** [comment]
Fixed by #253
